JavaScriptResource.mustache

```diff
@@ -110,7 +128,10 @@ fiftyoneDegreesManager = function() {
         return keyValuePairs;
     };
 
-    // Delete a cookie.
+    /**
+     * Delete a cookie.
+     * @param {string} name Cookie key
+     */
```

Comment:
if we are no longer deleting cookies when they are disabled, then why keep this method at all?  let's remove it altogether